### ISSUE TITLE
No sound on live games on Fire Tv

### ISSUE THREAD

**Issue by scottydulton** [maxgalbu/xbmc.plugin.video.nba#109]
i can get sound on both live games and archived games running kodi 18.9 Leia on pc, however on kodi 18.9 Leia on fire tv i cannot get sound on live games where i can in archived games. can anyone notice why with the 2 logs?

[kodi live game.log](https://github.com/maxgalbu/xbmc.plugin.video.nba/files/5753221/kodi.live.game.log)
[archive game.log](https://github.com/maxgalbu/xbmc.plugin.video.nba/files/5753222/archive.game.log)

**Comment by panpawel88** [comment]
@scottydulton as far as I understand you included logs for live and archive games when running the plugin on **Fire TV**. I think it will be easier to compare PC vs Fire TV logs for live games. Do you have the same addons installed on PC and Fire TV?

**Comment by scottydulton** [comment]
@panpawel88 yes mate i will upload pc logs tomorrow when live games are back on

**Comment by MRLB** [comment]
I am trying the latest version from @ivankokan and does not get audio too :-( (Fire TV 4K) (Kodi 18.9)

**Comment by ivankokan** [comment]
Yes, it is something weird about Fire TV.

**Comment by ivankokan** [comment]
@scottydulton @MRLB If you set Manual selection for **both** video and audio streams, how many of each you have to choose from, and which ones you pick? Do you experience issues with some streams of lower quality?

For example, running in Kodi on my PC for live Pacers - Cavaliers:
- Audio (1): English - AAC - 0 channels [default]
- Video (6): h264, 400000/800000/1600000/3000000/4500000/6000000 bps

**Comment by MRLB** [comment]
I've tried all 6 streams and all are without audio:-(

**Comment by ivankokan** [comment]
> I've tried all 6 streams and all are without audio:-(

How many audio streams to choose from?

**Comment by MRLB** [comment]
One. English AAC - 0 CHANNELS

**Comment by ivankokan** [comment]
It would be better to follow-up here: https://github.com/peak3d/inputstream.adaptive/issues/580.

**Comment by ivankokan** [comment]
@scottydulton @MRLB Nevertheless, can you test Audio only? IS.A settings -> Media -> Audio. Please provide Debug log for this. (I guess the previous state of Media setting was "All".)

**Comment by CamiloHT** [comment]
Hi same problem here using a xiaomi mibox s (Android) some tests with latest release (kodi and plugin)
Input Stream settings
All media --> only video 
Audio only --> only audio and it works

**Comment by scottydulton** [comment]
@CamiloHT have u got logs?

**Comment by ivankokan** [comment]
> Hi same problem here using a xiaomi mibox s (Android) some tests with latest release (kodi and plugin)
> Input Stream settings
> All media --> only video
> Audio only --> only audio and it works

That is something! Please comment again here: https://github.com/peak3d/inputstream.adaptive/issues/580. IS.A community will know better, and please provide Debug logs there.

**Comment by scottydulton** [comment]
@CamiloHT do you only have the problem in live games? or all games

**Comment by CamiloHT** [comment]
> @CamiloHT do you only have the problem in live games? or all games

Only Live games the same as you, Im looking how to get a log file

**Comment by CamiloHT** [comment]
[kodi.log](https://github.com/maxgalbu/xbmc.plugin.video.nba/files/5758434/kodi.log)
With live games I hope it helps

**Comment by ivankokan** [comment]
Guys, please switch to the opened IS.A issue, IS.A community will help better, especially considering this valuable discovery that Audio only works.

**Comment by scottydulton** [comment]
@panpawel88  here is the log from pc live games: https://paste.kodi.tv/uxiqenumah.kodi

and fire tv live games: 
[live game Fire TV.log](https://github.com/maxgalbu/xbmc.plugin.video.nba/files/5758442/live.game.Fire.TV.log)

**Comment by Jordan23M** [comment]
Just to mention, that i tried with Google Chromecast with Google TV, Kodi leia 18.9.

Probably has been mentioned previously, but ...

Live games Cleveland-Indiana, no sound, no available audio track to select, strangely buffering every 5 second, Audio stream only didn't start give an error (no file available), error for playing from beginning.

With OSD play detail info, shows no information about audio track, no codec, no info available.

Same device with official NBA app android TV works flawless, no problem, no buffering.

Archive games works well without problems.

Thanks for the great work, i hope can be fixed because Kodi has a great player and thanks to team for providing a great addons. 👍
